templates/terraform/sweeper_file.go.erb

```diff
@@ -31,22 +31,19 @@ func init() {
 // At the time of writing, the CI only passes us-central1 as the region
 func testSweep<%= sweeper_name -%>(region string) error {
 	resourceName := "<%= sweeper_name -%>"
-	log.Printf("[INFO] Sweeping %s",resourceName)
+	log.Printf("[INFO][SWEEPER_LOG] Starting sweeper for %s",resourceName)
 
-	config, err := sharedConfigForRegion(region)
-	if err != nil {
-		log.Printf("[INFO] error getting shared config for region: %s", err)
+	config, err := sharedConfigForRegion(region); if err != nil {
```

Comment:
nit: I'd prefer the original syntax instead of `; if`.

When statements are set up like the following, `x` is only defined in the block scope.

```
if x := foo(); x != nil {
  bar(x)
}
```

That isn't the case here, so there's no point in combining the lines. This is all just stylistic/syntax though, so no big deal either way.

Reply:
SGTM, I just thought it would save some lines